scripts/plotting/tem.py

```diff
@@ -74,11 +72,15 @@ def tem(adf):
     print(f"\t NOTE: redo_plot is set to {redo_plot}")
     #-----------------------------------------
 
-    #Location to saved TEM netCDF files
-    tem_loc = adf.get_basic_info("tem_loc")
+    #Grab TEM diagnostics options
+    tem_opts = adf.read_config_var("tem_info")
```

Comment:
I think we need a check here that if `tem_opts` is `None`, then the script returns with a warning message (otherwise users will run into an ADF-killing error on the line below).